chainer/backends/_chainerx.py

```diff
@@ -65,6 +65,10 @@ def __repr__(self):
         return '<{} {}>'.format(
             self.__class__.__name__, self.device.name)
 
+    def __str__(self):
+        return '{}: {}'.format(
+            self.__class__.__name__, self.device.name)
```

Comment:
Sorry it might not be clear, but the class name part is not needed: just return `self.device.name` in this case.